### ISSUE TITLE
Plugin Geomap: Tab order and focus issues

### ISSUE THREAD

**Issue by bsouster** [ServiceCanada/wet-boew#7]
Tabbing through the map, the canvas becomes highlighted and for several tabs, the focus is visually lost until it hits the GeoGratis link.
- fails 2.4.7 Focus Visible on most elements within canvas
- fails 2.4.3 Focus Order; tab order is literally all over the map
- I think it's tabbing through the point popup boxes even though they are in hidden state
